docs/UserGuide.md

```diff
@@ -374,20 +374,37 @@ Understanding the **command format** in Realodex is crucial for effectively mana
 
 **General Command Structure:**
 
-Commands typically follow this format: `COMMANDWORD PREFIX_ONE/UPPER_CASE PREFIX_TWO/UPPER_CASE ....` This structure helps in **executing specific actions** within the application.
+* Commands typically follow this format: `COMMANDWORD PREFIX_ONE/UPPER_CASE PREFIX_TWO/UPPER_CASE ....` This structure helps in **executing specific actions** within the application.
+* `COMMANDWORD` refers to the various commands user can input. 
+  * Examples include `add`, `delete`, `filter`. 
+  * A summary of all commands and their usages can be found in the [Commands Summary](#command-summary) table. 
+* Words in `UPPER_CASE` are the **parameters to be supplied by the user**.<br>
+    e.g. in `add n/NAME ....`, `NAME` is a parameter which can be used as `add n/John Doe ....`. 
+  * For more details on the parameters, do refer to the [Field Constraints](#field-constraints).
+* Parameters enclosed in `[]` are **optional to input**.<br>e.g. `[r/REMARK]` indicates an optional input field for that command.
+* You may input the parameters **in any order** (e.g. if the command specifies `n/NAME a/ADDRESS`, `a/ADDRESS n/NAME` is also acceptable).
 
 **Command Words:**
 
-`COMMANDWORD` refers to the **base commands** that the user can input, such as `add`, `delete`, `filter`. These commands are _case-insensitive_, meaning both `add` and `ADD` will perform the addition operation.
+* `COMMANDWORD` refers to the **base commands** that the user can input, such as `add`, `delete`, `filter`. These commands are _case-insensitive_, meaning both `add` and `ADD` will perform the addition operation.
 
 **Prefix Usage:**
 
-`PREFIX/` tags are used to **specify the type of data being entered**. For example, in `add n/John Doe p/98765432`, `n/` stands for name, and `p/` stands for phone number. Like command words, prefixes are also case-insensitive (`n/` and `N/` are treated the same).
+* `PREFIX/` tags are used to **specify the type of data being entered**. For example, in `add n/John Doe p/98765432`, `n/` stands for name, and `p/` stands for phone number. Like command words, prefixes are also case-insensitive (`n/` and `N/` are treated the same).
+* `PREFIX/` refers to the prefix tag associated with each user-written input. <br>
```

Comment:
```suggestion
* `PREFIX/` are used to **specify the type of data being entered**. For example, in `add n/John Doe p/98765432`, `n/` stands for name, and `p/` stands for phone number. Like command words, prefixes are also case-insensitive (`n/` and `N/` are treated the same).
* `PREFIX/` refers to the prefix associated with each user-written input. <br>
```

could consider not using the word tag as it might get confused with our tags

also a repeat of prefixes being case insensitive